tests/installation/installer_desktopselection.pm

```diff
@@ -34,6 +45,17 @@ sub run() {
     }
     assert_screen "$d-selected";
     send_key $cmd{next};
+
+    if (get_var('NEW_DESKTOP_SELECTION') && $d eq 'custom') {
+        assert_screen "pattern-selection";
+        for my $p (split(/,/, $install_patterns)) {
+            if (check_screen "pattern-$p", 5) {
```

Comment:
How about checking on both alternatives, already selected or not-selected with multi-tag assert screen and no custom timeout?